bungeecord/src/main/java/net/william278/huskchat/bungeecord/event/EventDispatcherBungee.java

```diff
@@ -0,0 +1,41 @@
+package net.william278.huskchat.bungeecord.event;
+
+import net.md_5.bungee.api.ProxyServer;
+import net.william278.huskchat.event.EventDispatcher;
+import net.william278.huskchat.event.IBroadcastMessageEvent;
+import net.william278.huskchat.event.IChatMessageEvent;
+import net.william278.huskchat.event.IPrivateMessageEvent;
+import net.william278.huskchat.player.Player;
+
+import java.util.ArrayList;
+import java.util.concurrent.CompletableFuture;
+
+public class EventDispatcherBungee implements EventDispatcher {
```

Comment:
Should probably be BungeeEventDispatcher in suiting with the plugin stylings (BungeeCommand, BungeeLogger etc.) Only exception to this rule is the main implementer; HuskChatBungee (for now)